controlplane/internal/imageregistry/image_registry_test.go

```diff
@@ -217,13 +226,13 @@ var _ = Describe("ImageRegistry Test", func() {
 		When("the user-provided image registry ConfigMap has an invalid toml configuration", func() {
 			It("returns parsing error and fails to create configmap", func() {
 				userRegistryCM := newUserProvidedRegistryCM(getInvalidRegistryToml(), certificate)
-				By("Calling the CreateImageRegistryConfigmap function")
-				imageRegistryConfigMap, err := CreateImageRegistryConfigmap(
+				By("Calling the GenerateImageRegistryConfigmap function")
+				imageRegistryConfigMap, err := GenerateImageRegistryConfigmap(
 					userRegistryCM,
-					namespace,
+					testNamespace,
 				)
 				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(Equal("(6, 13): cannot have two dots in one float"))
+				Expect(err.Error()).To(ContainSubstring("(6, 13): cannot have two dots in one float"))
```

Comment:
Can we see an example of descriptive error message here? Would this be `failed to parse image mirror registry toml for registry <name> : (6, 13): cannot have two dots in one float`?

Reply:
Yeah! Edited to match the error message